course_info/views.py

```diff
@@ -122,9 +130,9 @@ def _course_context(request, requested_keys, show_empty_fields=False, **kwargs):
             context['fields'].append(field)
 
     try:
-        school_info = ICommonsApi.from_request(request).get_school_info(course_info['course']['school_id'])
+        school_info = _api.get_school_info(course_info['course']['school_id'])
         school_title = school_info['title_long']
-    except KeyError:
+    except (ICommonsApiValidationError, KeyError):
```

Comment:
which is weird, because i ran unit tests before i committed and pushed.  anyway, fixed.